### PR TITLE
[IOTDB-4136] [sync] Sync sender-side statement generation

### PR DESCRIPTION
**Pull request by 23931017wu**
## Resolve SQL to statement

ShowPipeSinkStatment
ShowPipeSinkTypeStatement
ShowPipeStatament
OperatePipeSinkStatement
OperatePipeStatement
 
#### Before
图片

#### After
图片

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeSinkStatement.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import org.apache.iotdb.db.mpp.plan.analyze.QueryType;
+import org.apache.iotdb.db.mpp.plan.constant.StatementType;
+import org.apache.iotdb.db.mpp.plan.statement.IConfigStatement;
+import org.apache.iotdb.db.mpp.plan.statement.Statement;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class OperatePipeSinkStatement extends Statement implements IConfigStatement {
```
Comment (Cpaulyz) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeSinkStatement.java:32]: Split into CreatePipeSinkStatement, DropPipeSinkStatement

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeStatement.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import org.apache.iotdb.db.mpp.plan.analyze.QueryType;
+import org.apache.iotdb.db.mpp.plan.constant.StatementType;
+import org.apache.iotdb.db.mpp.plan.statement.IConfigStatement;
+import org.apache.iotdb.db.mpp.plan.statement.Statement;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class OperatePipeStatement extends Statement implements IConfigStatement {
```
Comment (Cpaulyz) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeStatement.java:32]: Split into CreatePipeStatement, StopPipeStatement, StartPipeStatement, DropPipeStatement

**Review by Cpaulyz — CHANGES_REQUESTED** [review]

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeSinkStatement.java (hunk 1 above)
Comment (23931017wu) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeSinkStatement.java]: OK ~

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeStatement.java (hunk 2 above)
Comment (23931017wu) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/OperatePipeStatement.java]: OK ~

**Review by Cpaulyz — APPROVED** [review]
LGTM

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**